### DISCUSSION TITLE
Wikidata edit-a-thon

### DISCUSSION THREAD

**Discussion by RVA2869** [govdirectory/website#399] [Ideas]
The idea is that every week we choose a country (existing/new) that we then supplement/add as much as possible in one week (the principle of an edit a thon😉)

Why: Because there are currently many 'inactive' countries.

**Comment by Abbe98** [comment]
I very much like this idea!

Makes me think of the Lexeme challange tool too, maybe something we could take inspiration from?

https://dicare.toolforge.org/lexemes/challenge.php

**Reply by Ainali** [reply]
Yes! I really like this idea too!

I think that one could select a country of focus each week and then the people showing up can decide on what exactly to collaborate on. Sometimes it might be finding sources, others creating queries or completing a subset of organizations that can get published.

A key for this is probably communication, getting the current week's focus in the Wikidata newsletter should be a focus.

**Reply by RVA2869** [reply]
Ah the classic gamification tactic 😄

**Reply by Ainali** [reply]
> A key for this is probably communication, getting the current week's focus in the Wikidata newsletter should be a focus.

By the way, if we want to start next week, we need to add a line to the newsletter very soon. It is usually sent on the Monday afternoon.

**Comment by Ainali** [comment]
What would be a good way to select the focus country each week? Should we just go alphabetically? Or should we base it on previous activity somehow? Or something wholly different (like countries with Wikimedia affiliates)?

And how far ahead should we plan? (This might be connected and resolved by itself after the first question is answered.)

**Reply by Ainali** [reply]
After have thinking about this a bit, I think alphabetical is better than any based on edits or statistics (which could lead to manipulative edits). However, for a bit more variation, I think we should rotate over the continents, in the order we have on the project page. As such, the first ten countries would be:

- Cameroon
- Argentina
- Armenia
- Albania
- Australia
- Central African Republic
- Bermuda
- India
- Austria
- New Zealand
- (and so on)

A downside I can see is that Oceania, with only two countries now, would reoccur every ten weeks, but I think we can live with that. what do you think

**Reply by Ainali** [reply]
Whichever way we choose, I suggest we mark on the project page with a symbol, perhaps [this](https://commons.wikimedia.org/wiki/File:Oxygen480-actions-view-calendar-week.svg), with a link to the idea of the focus week.

**Reply by Ainali** [reply]
> Whichever way we choose, I suggest we mark on the project page with a symbol, perhaps [this](https://commons.wikimedia.org/wiki/File:Oxygen480-actions-view-calendar-week.svg), with a link to the idea of the focus week.

I created [this template](https://www.wikidata.org/wiki/Wikidata:WikiProject_Govdirectory/Focus_marker) that can be included with `{{/Focus marker}}` after the country name in the list on the project page.

**Reply by Ainali** [reply]
We're now ten weeks in and no solution for Oceania. I don't know how we should solve this either, but suggest that we just skip Oceania for some time.

**Reply by RVA2869** [reply]
Skip for now, but next time we could focus on the States of Australia (in this case)

**Comment by RVA2869** [comment]
Or we could start with a cherry-picked country for which there is sufficient (official) data available and a reasonable user base.

**Reply by Ainali** [reply]
Sure, but having such a discussion each week seems like just creating more work for ourselves. 

Even when there is little official data already discovered, there's probably a lot we can do on-wiki thanks to the basic information available on the [general country subdivisions page](https://www.wikidata.org/wiki/Wikidata:WikiProject_Country_subdivision/Items) (besides also going hunting for good data sources).

**Reply by Abbe98** [reply]
An in between option is to go with cherry-picket countries but doing the cherry-picking in bulk, ~12 weeks at a time.

**Comment by RVA2869** [comment]
@Ainali I like the idea of ​​the continental cycle. When there are no (already made) countries left in a continent we could start a new one.

**Comment by Abbe98** [comment]
I think an important aspect of this would be to try to attract new contributors, outreach wise: pinging in local groups and newsletters, etc. But another aspect is to retain said contributors no matter if they want to contribute to other countries or not, that might be trickier.

**Reply by Ainali** [reply]
Yes, two different parts that might be tricky. But I think keeping the ordinary Collab hour open to anyone, regardless of the theme of the week, is a partial remedy (even though we rarely have people joining at all).

**Comment by Ainali** [comment]
FYI, I put [a note onwiki](https://www.wikidata.org/wiki/Wikidata_talk:WikiProject_Govdirectory#Themed_weeks) about this discussion.

**Comment by Ainali** [comment]
To not get stuck in the question of how to select, I suggest just getting started with [the list I made before](https://github.com/govdirectory/website/discussions/399#discussioncomment-9785349) making Cameroon the focus country next week.

If we feel this system doesn't work, we can always change it later.

**Comment by Ainali** [comment]
I created this page that can be included in various places: https://www.wikidata.org/wiki/Wikidata:WikiProject_Govdirectory/Weekly_collaboration

Right now, only included on https://www.wikidata.org/wiki/Wikidata:WikiProject_Govdirectory/Events

**Comment by RVA2869** [comment]
I started researching Cameroon's government structure... and this is going to be a tricky one.

**Reply by Ainali** [reply]
Please document any finds on-wiki (or dead ends on the wiki talk page).

**Comment by Ainali** [comment]
For outreach, what can we do? 

- I've added it to the [weekly tasks in the Wikidata newsletter](https://www.wikidata.org/wiki/Wikidata:Status_updates/Next#Weekly_Tasks).
- Posting about it on our Mastodon account is obvious.
- On the talk page for the country's Govdirectory project page (to have it show up on watch lists) 
- If there are WikiProjects on Wikidata related to the country, those should be notified.
- Relevant Telegram groups can be notified (like "African Wikimedians")

Any other ideas for this week or in general?

**Reply by RVA2869** [reply]
And the obvious one  `{{Ping project|Govdirectory}}`.

**Comment by Ainali** [comment]
First week was quite a step forward content wise. Cameroon's 1st and 2nd level are now 3 stars, and we can also query all ministries. Unfortunately, not a single of the regions or departments has any contact point, not even website, so I think it might be too early to publish.

**Comment by RVA2869** [comment]
With Australia, we have the first country with multiple governments. Related issue: #368

**Reply by Ainali** [reply]
Do you mean the states and the territories? That should be similar to UK and Scotland.

**Reply by RVA2869** [reply]
> That should be similar to UK and Scotland.

Not exactly the same but more like the USA and Germany. 
But for us (backend) it's the same 😅

**Comment by Ainali** [comment]
We are ten weeks in and should perhaps do a bit of an evaluation of how this initiative has been going. 

(I'll let others weigh in before adding my own impression.)

**Reply by Ainali** [reply]
Okay, so I'll start with my own impressions then. For me, this has mostly meant more administrative work, doing the update each Monday. I have not been getting inspiration often to do some work, only a few times when I have seen @RVA2869 active, but I think we have failed to inspire anyone else.

I don't know what else we can do. Perhaps we should this a whole month at a time? That would widen the window for people to join, and less often require the admin work. But perhaps something totally different is needed?

**Reply by Abbe98** [reply]
Do we think the failure to inspire others is caused by something else? Lack of outreach, bad documentation, etc?

**Reply by RVA2869** [reply]
I find it rather strange that there is a fairly large base of participants/interest (participants on WD:GD, Mastodon, Telegram (and Twitter/X)), but the ultimate “putting work into it” remains absent... 
Some possible reasons:

- Lack of good data.
- Don't know how to make a proper model.
- The country you want to work on is too complex (with multi-level governments, etc.).
- ...

FYI I discovered this project through the Wikipedia Weekly Network - LIVE Wikidata editing series; otherwise I would never have known about this project.

**Reply by Abbe98** [reply]
I would expect that to look similar in other Wikiprojects, it would be interesting to see to what extent challenges across Wikiproject match ours.

I think that better tooling could be a part of helping people to get started. Imagine AutoIdentifierInput but directly on Govdirectory.org or some type of instant feedback as you work on a country.

That said I think lack of data and scope overview might be trickier problems, Wikidata, the data model, etc we can get away from by exposing abstractions of those to editors but sources and scope not so much.
